js/everything.js

```diff
@@ -490,13 +508,17 @@ $(function() {
     // Handle form submission
     $("#form").submit(function(event) {
         event.preventDefault();
-
         var formValue = $.trim($("#v").val());
+        var formValueTime = formValue.match(/(&t=\d*)$/g);
```

Comment:
Actually, we should change the regex to be `/&t=(\d+)$/` since we only want to capture the digits. We can remove the global search as well since we just want the first value